tools/cdk-build-tools/tsconfig.json

```diff
@@ -2,19 +2,19 @@
   "compilerOptions": {
     "target": "ES2018",
     "module": "commonjs",
-    "lib": ["es2018"],
+    "lib": ["dom", "es2018"],
```

Comment:
add comment on why `dom` is needed.

Reply:
Oh - good you'd ask... It was needed in one stage of my experiments there but is no longer needed!